model-engine/model_engine_server/common/dtos/llms.py

```diff
@@ -143,6 +143,7 @@ class TokenOutput(BaseModel):
 
 class CompletionOutput(BaseModel):
     text: str
+    num_prompt_tokens: int
```

Comment:
i don't remember exactly, but i think having them optional could reduce duplicated returned data, since `num_completion_tokens` can be inferred, and `num_prompt_tokens` only need to be returned once